about.html

```diff
@@ -0,0 +1,163 @@
+    <!-- MAIN STARTS -->
+
+    <main>
+      <section class="main-opening about-page container">
+        <p class="intro about-hide">Adopting Global Best Practices in</p>
+        <h1 class="about-header">Community Pharmacy Practice</h1>
+        <hr />
+        <p class="description about-description">
+          Attended by most major manufacturers, distributors and community
+          Pharmcists in the Pharmacy space, the ACPN conference provides a
+          focused and unique opportunity to learn, share best practices, network
+          with peers, and interact with other healthcare providers in the
+          pharmaceutical space. At the ACPN Conference, community Pharmacists
+          can learn how best to offer pharmaceutical care, gain insights on more
+          efficient management processes and become aware of new technology and
+          techniques, which will ultimately help them and their companies: Sell
+          more products, Reduce costs, and Increase efficiencies
+        </p>
+        <p class="about-venue">
+          Please contact us via email for any further question and
+          clarifications about the ACPN conference 2023
+        </p>
+        <a class="email" href="#">info@acpn2023.org</a>
+      </section>
+    </main>
+    <!-- MAIN ENDS -->
+
+    <!-- SUB-SECTION STARTS -->
+    <section class="ending">
+      <div class="holder">
+        <h2>ACPN Conference 2023 Logo</h2>
+        <hr />
+        <p>
+          The Logo of the ACPN Conference 2023 will be decided through the
+          annual ACPN logo competition on June 20, 2023.
+        </p>
+        <h4 class="acpn">
+          ACPN NATIONAL <br />
+          CONFERENCE 2023.
+        </h4>
+      </div>
+      <hr class="dash" />
+      <div class="holder">
+        <h2>See the past ACPN Conference</h2>
+        <hr />
+        <p>
+          Take a look at the last two ACPN Conference which took place in Abuja,
+          Nigeria.
+        </p>
+
+        <div id="meeting">
+          <div class="meeting-start">
+            <h3 class="meeting-header">2021</h3>
+            <p class="meeting">ACPN Conference 2021 in Abuja</p>
+          </div>
+
+          <div class="meeting-middle">
+            <h3 class="meeting-header">2019</h3>
+            <p class="meeting">ACPN Conference 2021 in Enugu.</p>
+          </div>
+        </div>
+      </div>
+    </section>
+    <!-- SUB-SECTION ENDS -->
+
+    <!-- PARTNER STARTS -->
+    <section class="partner sponsor" id="sponsor">
+      <div class="container">
+        <h4>Partners</h4>
+        <hr />
+        <div class="logos logo-home">
+          <p class="sponsor1">daumkakao</p>
+          <img src="images/coke.png" alt="" />
+          <img src="images/facebook.png" alt="" />
+          <img src="images/amazon.png" alt="" />
+          <img src="images/mozilla.png" alt="" />
+          <img src="images/google.svg" alt="" />
+        </div>
+      </div>
```

Comment:
- [ ] As suggested by the last reviewer, kindly use the `ul` tag to wrap the image lis```
ting here, with each wrapped in a `li` tag. Also, you can use a `hx` or `span` got the sponsor class instead of a `p` tag, as that's not semantically correct

## Sample code:
 
```<div class="logos logo-home">
  <ul class="sponsor-list">
    <li class="sponsor-item"><span class="sponsor1">daumkakao</span></li>
    <li class="sponsor-item"><img src="images/coke.png" alt="" /></li>
    <li class="sponsor-item"><img src="images/facebook.png" alt="" /></li>
    <li class="sponsor-item"><img src="images/amazon.png" alt="" /></li>
    <li class="sponsor-item"><img src="images/mozilla.png" alt="" /></li>
    <li class="sponsor-item"><img src="images/google.svg" alt="" /></li>
  </ul>
</div>
```